mundus/interim_report_2023_aug_22.md

```diff
@@ -0,0 +1,351 @@
+
+| asset                                                                                                                                      | source                                                                                                                                     |
+| ------------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| <span class="hm">&#9888;</span> [0xBBeB516fb02a01611cBB...](https://explorer.zksync.io/address/0xBBeB516fb02a01611cBBE0453Fe3c580D7281011) | [0xe99FFA17f20F3f802286...](https://explorer.zksync.io/address/0xe99FFA17f20F3f8022862d1BD13519D305eF1377) <span class="hm">&#9888;</span> |
+| <span class="hm">&#9888;</span> [0x503234F203fC7Eb888EE...](https://explorer.zksync.io/address/0x503234F203fC7Eb888EEC8513210612a43Cf6115) | [0xCf58E8e67F2BcDd977e6...](https://explorer.zksync.io/address/0xCf58E8e67F2BcDd977e61bB6FDC1B0EEd6E1939d)                                 |
+| <span class="hm">&#9888;</span> [0x5D83C0850570De35eAF5...](https://explorer.zksync.io/address/0x5D83C0850570De35eAF5c9D6215BF2e8020f656B) | [0x071Bf614bc2c50140c1f...](https://explorer.zksync.io/address/0x071Bf614bc2c50140c1f094346774e529571A9Fb) <span class="hm">&#9888;</span> |
+| <span class="hm">&#9888;</span> [0x3355df6D4c9C3035724F...](https://explorer.zksync.io/address/0x3355df6D4c9C3035724Fd0e3914dE96A5a83aaf4) | [0x75D018f04f9cb3793653...](https://explorer.zksync.io/address/0x75D018f04f9cb37936530F7e3A909474565A2467)                                 |
+| <span class="hm">&#9888;</span> [0x5AEa5775959fBC2557Cc...](https://explorer.zksync.io/address/0x5AEa5775959fBC2557Cc8789bC1bf90A239D9a91) | [0x517F9cd13fE63e698d04...](https://explorer.zksync.io/address/0x517F9cd13fE63e698d0466ad854cDba5592eeA73)                                 |
+| <span class="hm">&#9888;</span> [0x493257fD37EDB34451f6...](https://explorer.zksync.io/address/0x493257fD37EDB34451f62EDf8D2a0C418852bA4C) | [0xCf58E8e67F2BcDd977e6...](https://explorer.zksync.io/address/0xCf58E8e67F2BcDd977e61bB6FDC1B0EEd6E1939d)                                 |
+
+## 2. ACLManager
+
+| issue # | issue<br />type       |
+| ------- | --------------------- |
+| 1       | EOA                   |
+| 2       | out of scope contract |
+| 3       | Unset values          |
+
+1. EOA [0xb76f765a785eca438e1d95f594490088afaf9acc](https://explorer.zksync.io/address/0xb76f765a785eca438e1d95f594490088afaf9acc) possesses the following roles:
+
+- `DEFAULT_ADMIN`
+- `EMERGENCY_ADMIN` -- RECHECK FIXED, 0xb76 -> EMERGENCY_ADMIN -> FALSE
+- `POOL_ADMIN`
+
+2. Out of scope contract [0x18F21fE46470F668cE72391Bb870A1822703a4fA](https://explorer.zksync.io/address/0x18F21fE46470F668cE72391Bb870A1822703a4fA) possesses the following roles:
```

Comment:
We will remove this contract. It most likely would have been a reserve helper contract.